docs/api_docs/python/README.md

```diff
@@ -2,7 +2,7 @@
 
 Documentation for the Python API is built with Sphinx under this directory.
 When new versions are released, the documentation is generated and published
-to the website.
+to [Read the Docs](https://readthedocs.org/projects/iree-python-api/).
```

Comment:
Ah, we can include this link _and_ https://iree-python-api.readthedocs.io/en/latest/.

* Users should go to https://iree-python-api.readthedocs.io/en/latest/
* Developers working on the docs can see the configuration, build logs, etc. at https://readthedocs.org/projects/iree-python-api/

Reply:
We link to https://iree-python-api.readthedocs.io/en/latest/ at https://github.com/iree-org/iree/blob/main/docs/website/docs/reference/bindings/python.md#usage. So I think it should be fine to link to the config here?

Reply:
But it is also fine for me to add a link to https://iree-python-api.readthedocs.io/en/latest/ here. I have some more doc improvements on the list for a follow up anyway.

Reply:
Adding more developer docs is cheap and can save future confusion. See the README for https://github.com/iree-org/iree/tree/main/docs/website as an example